tests/test_mtaosCsc.py

```diff
@@ -604,6 +618,70 @@ async def test_interruptWEP(self):
             with self.assertRaises(salobj.AckError):
                 await run_wep_task
 
+    def test_stress_below_limit(self):
```

Comment:
It doesn't look like the next couple tests belongs here, or, at least, they need some work.

This file here contains tests for the CSC so we should make sure we create an instance of the CSC and we manipulate the CSC to test some behavior. This starts by adding 

```
        async with self.make_csc(
            initial_state=salobj.State.STANDBY,
            config_dir=TEST_CONFIG_DIR,
            simulation_mode=0,
        ):
            ...
```

Then, you want to send a command to the CSC that would test the behavior you want. We now have the `offsetDoF` which probably can be used for what you want here.

it will look something like:

```
await self.remote.cmd_offsetDof.set_start(value=dof_aggr, timeout=STD_TIMEOUT)
``` 

Then you need to check the evens that where published as a result

```
dof = await self.assert_next_sample(self.remote.evt_degreeOfFreedom)
...
```